news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Community Mentorship Program Winter 2017 Signups Now Open"
+date: 2017-10-08 12:00:00 +0000
+---
+Have you ever opened up the editor and just looked at everything it offers in confusion? Do you have no idea where to even begin beatmapping? Worry no further, the Community Mentorship Program might be the place for you!
+
+![](/wiki/shared/news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open/banner.png)
+
+Since its inception in July of 2016, the program has gone through five cycles where experienced mappers teach aspiring new mappers everything there is to know about beatmapping, from the fundamentals of handling the editor to abstract and complex concepts of beatmap design, such as structure, flow, and many more!
+
+**This time around we are scouting for mentors and mentees for the game modes osu!, osu!catch and osu!taiko!** 
+
+This iteration of the program will be hosting three out of the four gamemodes of osu! - if you're interested here's a short timeline on this:
+- Mentor signups are open from 8th October 2017 until the 15th October 2017.
+- Mentee signups will open on the 22nd October 2017 and close on the 1st November 2017.
+- The 2017 Winter cycle will start on the 5th November 2017 and end on the 28th January 2018.
+
+During this time, the internal organization team will host various events on the server, such as assorted types of mapping contests, mapping discussions ([we also publish them on the forums as guides](https://osu.ppy.sh/community/forums/topics/514122)), or even just things like multiplayer lobbies with integrated feedback sessions!
```

Comment:
L19 might need a little rephrasing on this sentence
```
 or even just things like multiplayer lobbies with integrated feedback sessions!
 ^^^^^^^^^^^^^^^^^^^^^^^^^
````

Reply:
okay, but why
and what do you suggest

Reply:
@Okorin well, I ain't spoonfeeding suggestions anymore, you should find a way to mitigate that